src/_guides/libraries/library-tour.md

```diff
@@ -1293,6 +1293,60 @@ await Future.wait([
 print('Done with all the long steps!');
 ```
 
+`Future.wait()` returns a future which will complete once all the provided
+futures have completed, either with their results, or with an error if any of
+the provided futures fail. 
+
+#### Handling errors for multiple futures
+
+You can also wait for parallel operations on an [iterable]({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/FutureIterable/wait.html)
+or [record]({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/FutureRecord2/wait.html)
+of futures.
+
+These extensions return a future with the resulting values of the provided futures.
+Unlike `Future.wait()`, if any future in the collection completes with an error,
+`wait` completes with a [`ParallelWaitError`][] that allows the caller to
+handle individual errors and dispose successful results if necessary:
+
+```dart
+void main() async {
+  Future<bool> delete() async =>  ...
+  Future<bool> copy() async =>  ...
+  Future<bool> causesError() async =>  ...
+
+  try {
+    await [delete(), copy(), causesError()].wait;    // List of futures
```

Comment:
This code is not using the result value. 
It's the first/only example of using `await iterable.wait`, so I think it should show a *full* use-case.

It might be the only example of using the `.wait` getter some people will see.

Consider:

  bool success;
  try {
    /// Wait for each future in list.
    var results = await [delete(), copy(), causesError()].wait;
    success = !results.contains(false);
  } on ParallelWaitError<List<bool?>, List<AsyncError?>> catch (e) {
    success = false;
    // print ... all the same prints.
  }
  if (success) print("Success");
```

Reply:
@lrhn Thanks for fixing this! Is the `success` part necessary? I realize it's adding extra clarification that the `try` block represents successfully waiting for all the futures, and that `ParallelWaitError` represents a failure, despite having results returned, but isn't that kind of implied by the nature of `try` anyway?

Reply:
Ok, I think I understand now. I'll change them to `Future<void>` so the example is at least realistic (though still simpler, imo). 

If you think the use case shown in the example you wrote is much more valuable, though, let me know and I'll use it. I just worry that readers might conflate some of the other stuff that's going on in that example (like using `contains()` or somewhat "unpacking" the results into a boolean variable) with the question of "how to use `wait` on an iterable".

Reply:
I think switching to `Future<void>` is fine. 
I expect "just tell me when they're done" to be the most common use-case for list-wait.

Having an _unknown_ number of same-typed futures, that you want to run in parallel, and still want all the values of, seems more like an edge case for a badly-written API than a common use-case.
(If you end up doing `var results = await ids.map((id) => server.fetch(id)).wait;`, I'd ask why the server API doesn't provide a `server.fetchMultiple(Iterable<Id>)` function to begin with.)

It'll still happen. A lot. But I have no doubt people doing that will already know `Future.wait`.